server/pkg/repo/user.go

```diff
@@ -128,6 +129,38 @@ func (repo *UserRepository) GetAll(sinceTime int64, tillTime int64) ([]ente.User
 	return users, nil
 }
 
+// GetSubscribedUsers will return the list of all subscribed.
+func (repo *UserRepository) GetSubscribedUsersWithoutFamily(duration string) ([]ente.User, error) {
+	rows, err := repo.DB.Query(`SELECT u.user_id, u.encrypted_email, u.email_decryption_nonce, u.email_hash, s.created_at 
+	FROM subscriptions s 
+	INNER JOIN users u ON s.user_id = u.user_id 
+	WHERE s.created_at >= $1
+	AND u.family_admin_id IS NULL 
```

Comment:
s.created_at >= $1 should be s.created_at <= $1 (fixed it in the suggested query) and also added check for deleted users.